service/organisation-service.js

```diff
@@ -46,6 +46,19 @@ async function deleteOrganisation (orgId) {
         })
 }
 
+async function findManufacturerByName (name) {
+    let url = `${ADMIN_BASE_API_URL}/organisations/byTypeAndName/Manufacturer/`+name
```

Comment:
`${ADMIN_BASE_API_URL}/organisations/byTypeAndName/Manufacturer/${name}` would be nicer